src/select-menu/src/SelectMenu.js

```diff
@@ -177,7 +181,11 @@ SelectMenu.propTypes = {
   /*
    * When true, menu closes on option selection.
    */
-  closeOnSelect: PropTypes.bool
+  closeOnSelect: PropTypes.bool,
+
+  itemRenderer: PropTypes.func,
```

Comment:
Do you mind adding some comments here so that they get picked up by docgen (and show up on the docs site)?

Reply:
yep done